language/de/acp/extensions.php

```diff
@@ -122,6 +128,8 @@
 	'FORCE_UNSTABLE'					=> 'Immer auf instabile Entwicklungs-Versionen prüfen',
 	'EXTENSIONS_VERSION_CHECK_SETTINGS'	=> 'Einstellungen für Versions-Prüfung',
 
+	'BROWSE_EXTENSIONS_DATABASE'			=> 'phpBB-Erweiterungsdatenbank durchsuchen',
```

Comment:
Ich bin da unentschlossen. 

Es macht die Link-Liste bei den Extensions ziemlich breit, was bei schmalen Fenstern blöd aussieht. Ich finde es aber auch etwas unglücklich dass da ein externer Link neben zwei internen steht, da ist das zur Abgrenzung vielleicht etwas deutlicher. Bei den Sprachpaketen und Styles ist es dagegen der einzige Link dort oben, da finde ich das nicht so schlimm. Mir wäre sofort klar, dass es mit dem Link zur phpBB.com Datenbank geht. Ein unerfahrener User denkt aber vielleicht, dass das ein in phpBB integriertes Tool ist. Wenn ich noch mal drüber nachdenke ändert die jetzige Bezeichnung da aber vermutlich auch nichts dran, also habe ich auch nichts dagegen, das zu entfernen.

Also nehme ich das Ding nachher wenn ich zuhause bin raus und aktualisiere noch mal das Datum.